elements/pl-matrix-input/pl-matrix-input.mustache

```diff
@@ -39,7 +39,7 @@
 </script>
 <span id="pl-matrix-input-submission-{{uuid}}">
     {{#label}}<span>{{label}}</span>{{/label}}
-    <a role="button" class="btn btn-light btn-sm" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}{{#raw_submitted_answer}}<hr><h5><small class='text-uppercase'>What you submitted</small></h5><pre>{{raw_submitted_answer}}</pre>{{/raw_submitted_answer}}">INVALID <span><i class="fa fa-question-circle" aria-hidden="true"></i></span></a>
+    <a role="button" class="btn btn-light btn-sm" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}{{#raw_submitted_answer}}<hr><h5><small class='text-uppercase'>What you submitted</small></h5><code class='user-output-invalid'>{{raw_submitted_answer}}</code>{{/raw_submitted_answer}}">INVALID <span><i class="fa fa-question-circle" aria-hidden="true"></i></span></a>
```

Comment:
I'm wondering whether it would be a good idea to show the same help text here as we show on the original input (or at least parts of it)?

For example: https://github.com/PrairieLearn/PrairieLearn/blob/master/elements/pl-number-input/pl-number-input.mustache#L105

I also wonder whether this help text would be easier for students to process if we gave more examples of both valid and invalid inputs? For `pl-number-input` we could say:

Example valid inputs: `5`, `-17`, `4.25e7`

Example invalid inputs: `2/3`, `6+9`, `4.25*10^7`

At the moment the help text is quite technical in its descriptions of the valid format. Maybe we could lead with examples and put the technical details towards the bottom (maybe in fine print)?